__init__.py

```diff
@@ -1,31 +1,30 @@
-# Copyright 2017, Mycroft AI Inc.
+# Copyright 2017 Mycroft AI, Inc.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#    http://www.apache.org/licenses/LICENSE-2.0
+#     http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from concurrent.futures import ThreadPoolExecutor
-import json
 import re
+import sys
```

Comment:
This import isn't used

Reply:
The spelling causes it to pronounce wikipedia correctly. Think peed vs ped. I'll remove the unused import.

Reply:
The pronounciation shouldn't be handled here as it will need to be different for different TTS engines. Mimic2 has a separate database to correct the pronunciation and mycroft-core has a generic list of corrections (as @jarbasal said below)